File: mainn.cpp
#include <iostream>

auto square(int x) -> int { return x * x; }
int main() {
    auto number{7};
    auto result = square(number);

    std::cout << "Число:" << number << std::endl;
    std::cout << "Квадрат:" << result << std::endl;

    return 0;
}

File: mainn.py
# Функция square с аннотациями типов
def square(x: int) -> int:
    """Вычисляет квадрат числа"""
    return x * x


def main():
    # Аннотации типов для переменных
    number: int = 7
    result: int = square(number)
    
    print(f"Число: {number}")
    print(f"Квадрат: {result}")


if __name__ == "__main__":
    main()